app/hydrator/services/create/stores/config-store.js

```diff
@@ -592,7 +596,11 @@ class HydratorPlusPlusConfigStore {
     if (this.GLOBALS.etlBatchPipelines.includes(this.state.artifact.name)) {
```

Comment:
Params of setRangeRecordsPreview() can be removed. Earlier i have added to consume these values from artifacts config.

{minRecordsPreview = this.HYDRATOR_DEFAULT_VALUES.minRecordsPreview, maxRecordsPreview = this.HYDRATOR_DEFAULT_VALUES.maxRecordsPreview}